src/sass/base/_base.scss

```diff
@@ -15,7 +15,7 @@ body {
 }
 
 a {
-  color: $color-link;
+  color: white;
```

Comment:
Logically this does make sense. However, good software practise says to avoid "hard-coding" any values. The reason we do this is whenever we think we need to change the color theme of our site, we should do this from one place, instead of having to go through each file and change `color: white`. 

If you notice the old code was `color: $color-link`. `$color-link` is a variable we have declared in another file (where we have declared all our site colors). The correct solution, in this case, would be to change the assigned value of `$color-link`. This variable can be found here: [_variables.scss](https://github.com/Software-For-Love/software-for-love-site/blob/master/src/sass/abstracts/_variables.scss). If the color white was decided by the team then change it to that, however instead of putting 'white' just put the hexcode so `FFFFFF`, since most of our colors are in that format.